app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState, useEffect, useContext } from "react";
+import ModalMenuContext from "@context/ModalMenu/ModalMenu";
+import "./Header.scss";
+import "@components/Header/Header.scss";
+import Button from "@components/ui/Button/Button";
+import Logo from "@assets/logo4.svg?react";
+import Ellipsis from "@assets/icons/ellipsis.svg?react";
+import Plus from "@assets/icons/plus.svg?react";
+import ChevronDown from "@assets/icons/chevron.svg?react";
+import BREAKPOINTS from "@utils/Consts/breakpoints";
+
+const Header = () => {
+  const modalMenuCtx = useContext(ModalMenuContext);
+
+  const [isVisible, setIsVisible] = useState<boolean>(false);
+
+  const mql = window.matchMedia(`(max-width: ${BREAKPOINTS["screen-md-max"]})`);
```

Comment:
lets create useMediaQuery hook which will look similar to this: https://usehooks-ts.com/react-hook/use-media-query because it's very probable that we are going to need the same logic in some other places too and we will have to repeat it over and over again - so it's perfect case for custom hook. Here is another example: https://usehooks.com/usemediaquery